advisories/github-reviewed/2022/05/GHSA-hxrm-9w7p-39cc/GHSA-hxrm-9w7p-39cc.json

```diff
@@ -116,7 +116,7 @@
     {
       "package": {
         "ecosystem": "NuGet",
-        "name": "Microsoft.AspNetCore.App.Runtime.linux-musl-x64"
+        "name": "Microsoft.AspNetCore.App.Runtime.linux-musl-arm64"
```

Comment:
I don't know why it changed this.

Reply:
Hi @Tratcher, thanks for the update! I updated "Microsoft.AspNetCore.Owin" to "Microsoft.AspNetCore.Http" with a fixed version of 2.1.22 and VVR of < 2.1.22, and ignored the other changes in the commit. Let me know if I got that wrong or missed anything that should be included!